content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -35,45 +35,39 @@ These are methods that only an object of its class can invoke.
 It is the class that defines what an object data field or variables and methods will be.
 An example of a class is a circle class.
 A circle has a radius from which we can create different objects of type circle.
-The following code snipet explains this idea.
+The following code snippet explains this idea.
  
 ```java
  
 public class Circle {
  
   //instance variable
  
-private double radius=2.5;
- 
-  //no-argument constructor
- 
-  public Circle(){
- 
-  }
- 
-  //argument constructor
- 
-  public Circle(double radius){
- 
-  this.radius = radius;
- 
-  }
- 
-  //method
- 
-  public double getArea(){
- 
-  return Math.PI*radius*radius;
- 
-  }
- 
-  //method
- 
-  public  double getPerimeter(){
- 
-  return Math.PI*(2*radius);
- 
-  }
+    private double radius=2.5;
+    
+    //no-argument constructor
+    
+    public Circle(){
+    
+    }
+    
+    //argument constructor
+    
+    public Circle(double radius){
+     this.radius = radius;
```

Comment:
```java
this.radius = radius
```

it looks like you put a space in front of this line of code instead of an indent. Please fix it so it's indented like so:

```java
public Circle(double radius){
    this.radius = radius;
}
```

Reply:
Done